src/jobs/models/database/sequelize/sequelizeConnector.js

```diff
@@ -182,9 +182,9 @@ async function initSchemas() {
         }
     });
 
-    job.webhook = job.hasMany(webhook);
+    // job.webhook = job.hasMany(webhook);
```

Comment:
uncomment

Reply:
done, added jobs and webhooks N:M association